discordbot.py

```diff
@@ -1,11 +1,26 @@
 import asyncio
+import string
 import discord
 from config import SG_TOKEN, DISCORD_TOKEN
 import requests
 from discord.ext.commands import Bot
 
+
+def sanitize_repo_name(repo_name):
+    allowed_chars = set(string.ascii_letters + string.digits + "./")
```

Comment:
This is actually great 🎉 was top on my mind today to fix this! 

Additionally, it would be really good to follow the official github regex pattern listed [developer repo](https://sourcegraph.com/github.com/github-developer/creating-ci-tests-with-the-checks-api/-/blob/server.rb?L45) that includes `-` and `_` 

Do you think can we make allowlist for host ['github.com', 'bitbucket.com', 'gitlab.com'] further prevent from feeding other domain urls ? let me know your opinion 😄 

Thanks again!